.devcontainer/docker-compose.yml

```
@@ -0,0 +1,68 @@
+    container_name: database_dozens
+    restart: unless-stopped
+    networks:
+      - dozens-ecosystem
+    ports:
+      - '5432:5432'
+    environment:
+      - POSTGRES_USER=dozens
+      - POSTGRES_PASSWORD=dozens
+      - POSTGRES_DB=dozens
+    healthcheck:
+    healthcheck:
```

YAMLlint (1.37.1)

[error] duplication of key "healthcheck" in mapping

(key-duplicates)